packages/driver/test/cypress/integration/commands/assertions_spec.coffee

```diff
@@ -1019,6 +1019,12 @@ describe "src/cy/commands/assertions", ->
 
         expect(null).to.have.html("foo")
 
+      it "partial match", ->
+        expect(@$div).to.have.contain.html('button')
```

Comment:
Is this supposed to be `to.have.html` instead of `to.have.contain.html`?

Reply:
Works either way, but will fix